strings/tips/meta.txt

```diff
@@ -1,25 +1,24 @@
-This is a game that is constantly being developed for. Expect things to be added, removed, fixed, oudated and broken on a daily basis.
-In maps where there are water, you can simply run to rivers or areas where there are water to extinguish yourself while being set on fire.
-In the character setup screen, you can set your keybinds to control actions much easily.
-While alive, use 'resist' (Default: B) to stop drop and roll when on fire. Humans can also use the resist command to struggle out of nests while captured or to simply unbuckle from a chair.
+This is a game that is constantly being developed. Expect things to be added, removed, fixed, oudated and broken on a daily basis.
+In maps where there is water, you can simply run to rivers or areas where there is water to extinguish yourself while on fire.
+In the character setup screen, you can set your keybinds to control actions much more easily.
+While alive, use 'resist' (Default: B) to stop, drop and roll when on fire. Humans can also use the resist command to struggle out of warrior grabs or to simply unbuckle from a chair.
 Help Intent (Default: 1) allows you to exchange positions with another mob with their help intent active. Other intents (Disarm, Grab or Harm) will push mobs aside.
-When in doubt: Adminhelp for problems in common for people violating the rules, Mentorhelp for problems that you want to know.
+When in doubt: Adminhelp for problems with people violating the rules, Mentorhelp for questions that you want to know the answer to.
 Check the SS13 Webmap to learn the layout of the ship or the map currently in play.
-After you ghosted out of the game (either from dying, cryo or manual ghost), you can respawn back into the game after waiting for 30 minutes (as a human) or 2 minutes (as a xenomorph) if there is a larva.
+After you have ghosted out of the game (either from dying, cryo or manual ghost), you can respawn back into the game after waiting for 30 minutes (as a marine) or if there is a larva available (as a xenomorph).
 To sing, add "%" before any text. Singing on radios will not work, but singing on intercomms will.
 You can respawn during End-Of-Round Deathmatch in the OOC tab.
-Sundering is a concept that reduces the effectiveness of xeno armor by percentage. Xenomorph drones can heal and decrease sunder. Resting heals sunder, but slowly.
+Sundering reduces the effectiveness of xeno armor by percentage. Some xenomorph castes can heal and decrease sunder. Resting heals sunder, but slowly.
 When you want to claim a mob, claim it now! Others will claim it as soon as they are available.
 You can customize your ghost using the Ghost Customization command under the Preferences tab.
-As either a marine or a xeno, you will die quickly and horribly if you find yourself in a pressureless enviroment (a.k.a spess or water).
+As either a marine or a xeno, you will die quickly and horribly if you find yourself in a pressureless enviroment (a.k.a spess or underwater).
```

Comment:
Where do we even have underwater?

Reply:
real. why was water included in the tip in the first place. we may never know.